styles/globals.css

```diff
@@ -13,12 +13,28 @@ body {
   background-color: #f8f9fa;
 }
 
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
+h1 {
+  @apply text-4xl;
+  @apply font-bold;
+  @apply tracking-tight;
```

Comment:
nit: we can make the code dry by keeping the `tracking-tight` class as it was before here

```javascript
h1,
h2,
h3,
h4
{
 @apply tracking-tight;
}
```